spec/super_good/solidus_taxjar/backfill_transactions_spec.rb

```diff
@@ -2,11 +2,10 @@
 
 RSpec.describe SuperGood::SolidusTaxjar::BackfillTransactions do
   describe ".call" do
```

Comment:
I know it's not in this PR but this should be `describe "#call" do` because it's an instance method. That being said since we aren't passing anything to the initializer anymore it could also just be a class method instead.

Reply:
Addressed!